python/ray/data/_internal/arrow_block.py

```diff
@@ -658,9 +646,7 @@ def aggregate_combined_blocks(
                 if next_row is None:
                     next_row = next(iter)
                 next_keys = key_fn(next_row)
-                next_key_names = (
-                    next_row._row.schema.names[: len(keys)] if key is not None else None
-                )
+                next_key_names = keys
```

Comment:
nit: i think there is only one usage of this variable, so we can drop it and use `keys` directly.

Reply:
ok, I renamed it to something more clear so that there's better readability